### PR TITLE
Proposed charter changes from RFC

### PR DESCRIPTION
**Pull request by amzn-changml**
As per the RFC in https://github.com/o3de/sig-security/issues/2

This is editable for anyone in the maintainers group, so feel free to pull this and add changes as needed.

### CODE REVIEW

File: governance/SIG Security Charter.md [hunk 1]
```diff
@@ -1,97 +1,73 @@
...
 
-## Individual Contributors
-Must provide links and examples to security flaws, discoveries, or potential issues.
+**Individual Contributors**
 
 Additional information not found in the sig-governance related to contributors.
 
-## Maintainers
-Must validate and verify that submitted issue or flaw exists and should be accepted into practices.
+**Maintainers**
```
Comment (Kadino) [inline governance/SIG Security Charter.md:58]: Removing the expected role activities makes this weaker, but may be more accurate to the current state of the SIG guidance (I've been unable to attend meetings, and am unaware)

**Review by Kadino — APPROVED** [review]
This is a good start!

File: governance/SIG Security Charter.md [hunk 2]
```diff
@@ -1,97 +1,73 @@
...
 ## Overview of SIG
-A central knowledge advisory group that provides resources and information security compliant, resolution and XYZ.
-
-All about security of systems and infrastructure resources. Security patching, resources and infra up to date.
-Provide the basic services of APPSEC related to this project.
-
-Two concise lines explaining what this SIG does with bullet points of the major responsibilities
-
-- Responsibility
+SIG Security is a special interest group for O3DE code security that works as a central knowledge advisory group and  provides resources, consultation, reporting mechanisms  and information for security compliance, and security related issue resolution. This SIG also helps in keeping security patching up to date, and provide the basic services related to  APPSEC (Application Security) of O3DE.
```
Comment (lmbr-pip) [inline governance/SIG Security Charter.md:4]: nit: "is a special interest group for O3DE security" - remove code?

nit: "reporting mechanisms and" - looks like two spaces between "mechanisms" and "and"

File: governance/SIG Security Charter.md [hunk 3]
```diff
@@ -1,97 +1,73 @@
...
-- Create and maintain mechanism and guidelines to raise concerns for security issues.
-- Create and maintain tools to do version checking to ensure libraries are in compliance.
-- Create and maintain SPDX scanning tools to detect violations
-- Provide and report on penetration testing on point release software before general availability.
-- ***Look into threat modelling and compliance***
-
-- Items that are the core responsibilities of SIG
+
+- Triage and Acceptance of incoming security issues.
+- Creating mechanisms for intake of private security issues from any source.
```
Comment (lmbr-pip) [inline governance/SIG Security Charter.md:18]: "Creation and maintenance of mechanisms"...?

File: governance/SIG Security Charter.md [hunk 4]
```diff
@@ -1,97 +1,73 @@
...
+- Management of GitHub security advisories for the O3DE repros.
+- Management of security reporting email and initial disclosure lists.
```
Comment (lmbr-pip) [inline governance/SIG Security Charter.md:20]: Maybe split and combine first part with first item and make "management of security disclosures" own item?

File: governance/SIG Security Charter.md [hunk 5]
```diff
@@ -1,97 +1,73 @@
...
+- Provides mechanisms for reporting security issues manually
```
Comment (lmbr-pip) [inline governance/SIG Security Charter.md:21]: This feels a repeat of first item.

File: governance/SIG Security Charter.md [hunk 6]
```diff
@@ -1,97 +1,73 @@
...
+- Provides tooling and automation for security issue identification and running campaigns to resolve identified security issues, including deprecation of software components.
```
Comment (lmbr-pip) [inline governance/SIG Security Charter.md:22]: Split into two.. "Automation of security issue detection"  and "running campagins across O3DE to resolve..."

File: governance/SIG Security Charter.md [hunk 7]
```diff
@@ -1,97 +1,73 @@
...
+- Provide SPDX (Software Package Data Exchange) scanning tools to detect violations.
 
 ## Cross-cutting Processes
 
 - Disseminating information to other sigs for action / update/ fixing
```
Comment (lmbr-pip) [inline governance/SIG Security Charter.md:27]: Maybe add something about driving dates and resources with impacted SIGs? SIG-Security does not own/provide resources for patching issues outside of code SIG-Security directly owns.

File: governance/SIG Security Charter.md (hunk 1 above)
Comment (lmbr-pip) [reply governance/SIG Security Charter.md:58]: I think the intent is to only call things out where this SIG diverges from standard role behaviours. 

I would add that we expect maintainers to be able to evaluate security issues and ensure they are correctly reported with sufficient information to accept. However we can add maintainer expectations once we have issue response runbook in place.

**Review by lmbr-pip — COMMENTED** [review]
LG. Thanks for doing this.

Did notice that charter is some inconsistent with use of '.' at end of sentences in the list sections.

File: governance/SIG Security Charter.md [hunk 8]
```diff
@@ -1,97 +1,73 @@
...
-- Major goals that SIG seeks to generally achieve
+- To be set after rest of the charter is reviewed.
 
 ## Scope
-Involve gathering information about security flaws. 
-Identify potential security risks/gaps in softare.
-Define best practices for ensuring secure code
-
-- Generalized overall scope of work
+-  Involves gathering information about security flaws.
```
Comment (myfoostrong) [inline governance/SIG Security Charter.md:11]: To match stance of other Scope items:
`-  Gather information about...`

**Review by myfoostrong — CHANGES_REQUESTED** [review]
Called out two items, and agree with the other comments already brought up

File: governance/SIG Security Charter.md (hunk 2 above)
Comment (allisaurus) [reply governance/SIG Security Charter.md:4]: I feel like this paragraph would be more readable if we simplified the sentence structure. What do you think about:

"SIG Security is a special interest group for O3DE security. It serves as an advisory group for security related issues like compliance, security issue resolution, and security patching. It's also the group responsible for maintaining vulnerability reporting mechanisms and assisting with other tasks that fall under the [Application Security (AppSec)](https://en.wikipedia.org/wiki/Application_security) umbrella of concern."

File: governance/SIG Security Charter.md [hunk 9]
```diff
@@ -1,97 +1,73 @@
...
-
-- Responsibility
+SIG Security is a special interest group for O3DE code security that works as a central knowledge advisory group and  provides resources, consultation, reporting mechanisms  and information for security compliance, and security related issue resolution. This SIG also helps in keeping security patching up to date, and provide the basic services related to  APPSEC (Application Security) of O3DE.
 
 ## Goals
 
-- Major goals that SIG seeks to generally achieve
+- To be set after rest of the charter is reviewed.
 
 ## Scope
```
Comment (allisaurus) [inline governance/SIG Security Charter.md:10]: As worded, I don't really understand the point of this section in addition to the below "In scope" section. It also reads more like a "Goals" section, esp. the 2 bullets re: Identifying potential risks in the code and defining best practices. Can we remove it, or is it part of some standard charter template?

File: governance/SIG Security Charter.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (amzn-changml) [reply governance/SIG Security Charter.md:4]: I can agree with that. @lmbr-pip @dshmz to commit in this and any other suggestions

File: governance/SIG Security Charter.md [hunk 10]
```diff
@@ -1,97 +1,73 @@
...
+- Provides tooling and automation for security issue identification and running campaigns to resolve identified security issues, including deprecation of software components.
+- Provide SPDX (Software Package Data Exchange) scanning tools to detect violations.
 
 ## Cross-cutting Processes
 
 - Disseminating information to other sigs for action / update/ fixing
+- Review and provide feedback on SIG presented features and implementation
 - Provide SPDX Licensing report violations to corresponding SIGs
-- Advise SIGs on security level of new libraries and add to published compliance list
+- Advise SIGs on security level of new libraries
```
[the remark this answers is not in the document]
Comment (amzn-changml) [reply governance/SIG Security Charter.md:30]:
```suggestion
- Advise SIGs on secure use of new libraries 
```

File: governance/SIG Security Charter.md (hunk 8 above)
Comment (amzn-changml) [reply governance/SIG Security Charter.md:11]:
```suggestion
-  Gather information about security flaws. 
```

File: governance/SIG Security Charter.md (hunk 9 above)
Comment (lmbr-pip) [reply governance/SIG Security Charter.md:10]: Its part of the standard O3DE charter template.

File: governance/SIG Security Charter.md (hunk 3 above)
Comment (lmbr-pip) [inline governance/SIG Security Charter.md:18]:
```suggestion
- Creation and maintenance of mechanisms for the secure intake of vulnerability issues, including any security reporting email lists for O3DE.
```

File: governance/SIG Security Charter.md (hunk 4 above)
Comment (lmbr-pip) [inline governance/SIG Security Charter.md:20]:
```suggestion
- Management of any security disclosure mechanisms including email lists.
```

File: governance/SIG Security Charter.md (hunk 5 above)
Comment (lmbr-pip) [inline governance/SIG Security Charter.md:21]:
```suggestion
```

File: governance/SIG Security Charter.md (hunk 6 above)
Comment (lmbr-pip) [inline governance/SIG Security Charter.md:22]:
```suggestion
- Assists in the selection or development of tooling and automation for security issue identification.
- Runs security campaigns to resolve identified security issues, including deprecation of software components.
```

**Review by lmbr-pip — COMMENTED** [review]
Added suggestions

File: governance/SIG Security Charter.md [hunk 11]
```diff
@@ -1,97 +1,73 @@
...
 ## Out of Scope
 - Not responsible for fixing any detected or reported security issues.
-- Not responsible for enforcing or acting upon any legal compliance issues
+- Not responsible for enforcing or acting upon any legal compliance issues.
 - Not responsible for implementing security or IP detection tools into automation or build chains, but may advise SIGs on where to implement.
-
 - Items that are optional or are not the responsibility of this SIG.
 
-## SIG Links and lists:
+**SIG Links and lists:**
```
Comment (lmbr-pip) [inline governance/SIG Security Charter.md:42]:
```suggestion
## SIG Links and lists
```

File: governance/SIG Security Charter.md [hunk 12]
```diff
@@ -1,97 +1,73 @@
...
-- Issues/PRs
-- Meeting agenda & Notes
+- Joining this SIG - https://github.com/orgs/o3de/teams/sig-security
+- Slack/Discord - https://discord.com/channels/805939474655346758/816043899477950475 
+- Mailing list - https://lists.o3de.org/g/sig-security
+- Issues/PRs - https://github.com/o3de/sig-security/issues/
+- Meeting agenda & Notes - https://github.com/o3de/sig-security/issues/1 
 
-## Roles and Organization Management
+**Roles and Organization Management**
```
Comment (lmbr-pip) [inline governance/SIG Security Charter.md:50]:
```suggestion
## Roles and Organization Management
```

File: governance/SIG Security Charter.md [hunk 13]
```diff
@@ -1,97 +1,73 @@
...
 
-SIG Docs adheres to the standards for roles and organization management as specified by <sig-governance>. This SIG opts in to updates and modifications to <sig-governance>
+SIG Security adheres to the standards for roles and organization management as specified by <sig-governance>. This SIG opts in to updates and modifications to <sig-governance>
 
-## Individual Contributors
-Must provide links and examples to security flaws, discoveries, or potential issues.
+**Individual Contributors**
```
Comment (lmbr-pip) [inline governance/SIG Security Charter.md:54]:
```suggestion
##I# ndividual Contributors

Should provide steps to reproduce, links and examples for security flaws, discoveries, or potential issues. Should raise potential security issues through the [appropriate mechanisms](https://github.com/o3de/sig-security/blob/main/rfcs/rfc-sec-20220215-1.md)
```

File: governance/SIG Security Charter.md (hunk 1 above)
Comment (lmbr-pip) [inline governance/SIG Security Charter.md:58]:
```suggestion
###Maintainers

* Run issue triage when SIG-Chair(s) are unavailable. Must validate and verify that submitted issue or flaw exists and should be accepted by SIG.
* Help the SIG stay on top of any security concerns:
    * Engage with the [SIR team process](https://github.com/o3de/sig-security/blob/main/rfcs/rfc-sec-20220316-1.md)
    * (Optionally) Monitor GitHub issues for any security impacting issues and raise with SIG.
    * (Optionally) Monitor DIscord for any security impacting issues and raise with SIG.

```

File: governance/SIG Security Charter.md [hunk 14]
```diff
@@ -1,97 +1,73 @@
...
 Additional information not found in the sig-governance related to contributors.
 
-## Maintainers
-Must validate and verify that submitted issue or flaw exists and should be accepted into practices.
+**Maintainers**
 
 Additional information not found in the sig-governance related to contributors
 
-## Additional responsibilities of Chairs
+**Additional responsibilities of Chairs**
```
Comment (lmbr-pip) [inline governance/SIG Security Charter.md:62]:
```suggestion
### Additional responsibilities of Chairs
```

File: governance/SIG Security Charter.md [hunk 15]
```diff
@@ -1,97 +1,73 @@
...
 
 Additional information not found in the sig-governance related to SIG Chairs
 
-## Subproject Creation
+**Subproject Creation**
```
Comment (lmbr-pip) [inline governance/SIG Security Charter.md:66]:
```suggestion
## Subproject Creation
```

File: governance/SIG Security Charter.md [hunk 16]
```diff
@@ -1,97 +1,73 @@
...
 
 Additional information not found in the sig-governance related to subproject creation
 
-## Deviations from sig-governance
+**Deviations from sig-governance**
```
Comment (lmbr-pip) [inline governance/SIG Security Charter.md:70]:
```suggestion
## Deviations from sig-governance
```

**Review by lmbr-pip — COMMENTED** [review]
Final suggestions

File: governance/SIG Security Charter.md (hunk 1 above)
Comment (lmbr-pip) [reply governance/SIG Security Charter.md:58]: Fixed below. Resolving.

File: governance/SIG Security Charter.md (hunk 3 above)
Comment (lmbr-pip) [reply governance/SIG Security Charter.md:18]: Added suggestion below.

**Review by lmbr-pip — APPROVED** [review]

**Review by allisaurus — APPROVED** [review]

**Pull request merged**